test/test_other.py

```diff
@@ -3298,13 +3298,16 @@ def test_embind_tsgen_memory64(self):
     self.assertFileContents(test_file('other/embind_tsgen_memory64.d.ts'), read_file('embind_tsgen_memory64.d.ts'))
 
   def test_embind_tsgen_exceptions(self):
-    # Check that when Wasm exceptions and assertions are enabled bindings still generate.
-    self.run_process([EMXX, test_file('other/embind_tsgen.cpp'),
-                      '-lembind', '-fwasm-exceptions', '-sASSERTIONS',
-                      # Use the deprecated `--embind-emit-tsd` to ensure it
-                      # still works until removed.
-                      '--embind-emit-tsd', 'embind_tsgen.d.ts', '-Wno-deprecated'])
-    self.assertFileContents(test_file('other/embind_tsgen.d.ts'), read_file('embind_tsgen.d.ts'))
+    for wasm_exnref in [0, 1]:
```

Comment:
Can you use `@parameterize` here instead of the loop?

Reply:
Something like:

```
@parameterize({
   '', [0],
   'wasm_exnref', [1],
})
```

Reply:
Done: https://github.com/emscripten-core/emscripten/pull/21913/commits/db345b2da807de76d4eabff8559413ffd598d257